package.json

```diff
@@ -170,6 +170,12 @@
 		"env:logs": "node ./tools/local-env/scripts/docker.js logs",
 		"env:pull": "node ./tools/local-env/scripts/docker.js pull",
 		"test:php": "node ./tools/local-env/scripts/docker.js run --rm phpunit phpunit",
+	  	"test:php_clover": "node ./tools/local-env/scripts/docker.js run --rm coverage phpunit --coverage-clover coverage/clover",
+	  	"test:php_crap4j": "node ./tools/local-env/scripts/docker.js run --rm coverage phpunit --coverage-crap4j coverage/crap4j",
+	  	"test:php_htm": "node ./tools/local-env/scripts/docker.js run --rm coverage phpunit --coverage-html coverage/html",
+	  	"test:php_php": "node ./tools/local-env/scripts/docker.js run --rm coverage phpunit --coverage-php coverage/php/index.php",
+	  	"test:php_text": "node ./tools/local-env/scripts/docker.js run --rm coverage phpunit --coverage-text coverage/text",
+	  	"test:bash": "node ./tools/local-env/scripts/docker.js run --rm phpunit bash",
```

Comment:
Indention is off here, it's a mixture of tabs and spaces, it should be 1 single tab for each line, also potentially removing some of these per feedback above